wifiphisher/extensions/deauth.py

```diff
@@ -181,5 +184,5 @@ def send_channels(self):
         :rtype: list
         """
         if self._is_frenzy:
-            return [str(k) for k in range(1, 14)]
+            return [str(k) for k in constants.ALL_2G_CHANNELS]
```

Comment:
I didn't mean why it's string 😄 . Rather why add all the channels when we can be more efficient and add the channels we need. I'm saying this because inside the `get_packet` we already add the channel depending on the mode so why add everything.